spec/javascript/components/molecules/UnitDropdown.test.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { mount, shallow } from 'enzyme'
+import Select from 'react-select'
+
+import UnitDropdown, { renderUnitOption } from '~/components/molecules/UnitDropdown'
+
+const ON_CHANGE = jest.fn()
+const units = [
+  {
+    id: '1',
+    unit_number: '123',
```

Comment:
Can we add an example with unit_number = '0'? It's unlikely but doesn't seem impossible, just want to make sure this doesn't break